pkg/apiserver/filters/auditing.go

```diff
@@ -0,0 +1,44 @@
+package filters
+
+import (
+	"k8s.io/klog"
+	"kubesphere.io/kubesphere/pkg/apiserver/auditing"
+	"kubesphere.io/kubesphere/pkg/apiserver/request"
+	"net/http"
+)
+
+func WithAuditing(handler http.Handler, a auditing.Auditing) http.Handler {
+
+	return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+
+		if !a.Enable() {
```

Comment:
it's unnecessary to do a check here, just return `handler` when auditing is not enabled. Check out the code above